test/python/circuit/test_gate_definitions.py

```diff
@@ -115,6 +117,74 @@ def test_cx_definition(self):
         self.assertTrue(Operator(circ).equiv(Operator(decomposed_circ)))
 
 
+@ddt
+class TestStandardGates(QiskitTestCase):
+    """Standard Extension Test."""
+
+    @unpack
+    @data(
+        *inspect.getmembers(
+            standard_gates,
+            predicate=lambda value: (inspect.isclass(value)
+                                     and issubclass(value, Gate)))
+    )
+    def test_definition_parameters(self, class_name, gate_class):
+        """Verify definitions from standard library include correct parameters."""
+
+        free_params = _get_free_params(gate_class)
+        n_params = len(free_params)
+        param_vector = ParameterVector('th', n_params)
+
+        if class_name in ('MCPhaseGate', 'MCU1Gate'):
+            param_vector = param_vector[:-1]
+            gate = gate_class(*param_vector, num_ctrl_qubits=2)
+        elif class_name in ('MCXGate', 'MCXGrayCode', 'MCXRecursive', 'MCXVChain'):
+            num_ctrl_qubits = 2
+            param_vector = param_vector[:-1]
+            gate = gate_class(num_ctrl_qubits, *param_vector)
+        elif class_name == 'MSGate':
+            num_qubits = 2
+            param_vector = param_vector[:-1]
+            gate = gate_class(num_qubits, *param_vector)
+        else:
+            gate = gate_class(*param_vector)
+
+        if gate.definition is not None:
+            self.assertEqual(gate.definition.parameters, set(param_vector))
+
+    @unpack
+    @data(
+        *inspect.getmembers(
+            standard_gates,
+            predicate=lambda value: (inspect.isclass(value)
+                                     and issubclass(value, Gate)))
+    )
+    def test_to_matrix(self, class_name, gate_class):
```

Comment:
This seems similar to `test_to_matrix_op` in test_extensions_standard. Perhaps just keep one?

Reply:
Good catch, removed in 7be5b9f.